vitrivr-engine-module-fes/src/main/kotlin/org/vitrivr/engine/base/features/external/common/ApiWrapper.kt

```diff
@@ -0,0 +1,323 @@
+package org.vitrivr.engine.base.features.external.common
+
+import okhttp3.OkHttpClient
+import org.openapitools.client.apis.*
+import org.openapitools.client.models.*
+import org.vitrivr.engine.core.model.content.element.AudioContent
+import org.vitrivr.engine.core.util.extension.toDataURL
+import java.awt.image.BufferedImage
+
+data class JobResult<S>(
+        val status: JobState,
+        val result: S?
+)
+
+class JobWrapper<T, S>(
```

Comment:
Why don't you have these in separate files? If these are internal classes, mark them so.